src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -60,107 +65,119 @@
 using namespace chip;
 using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::Groups;
+using namespace chip::Credentials;
 
-static bool isGroupPresent(EndpointId endpoint, GroupId groupId);
-
-static bool bindingGroupMatch(EndpointId endpoint, GroupId groupId, EmberBindingTableEntry * entry);
+static FabricIndex GetFabricIndex()
+{
+    return emberAfCurrentCommand()->source->GetSessionHandle().GetFabricIndex();
+}
 
-static uint8_t findGroupIndex(EndpointId endpoint, GroupId groupId);
+static bool isGroupIdValid(chip::GroupId id)
+{
+    return id > 0;
```

Comment:
@bzbarsky-apple  I wished the Universal Group ID's were never introduced. Then the group concept as a Data Model thing could use 1 - 0xffff, and use '0' for special cases like in Scenes. Do nodes 'self assign' them to the Universal Groups? What key material is used? Not understanding how this concepts works, makes it difficult to answer your question.

Reply:
In now read in the spec on Universal Group ID:
_The Commissioner SHALL configure one or more shared keys for these groups on all Nodes within
the Fabric. Because the keys and IPv6 multicast prefixes are different across Fabrics, Universal
Groups only enable communication within a specific Fabric._

So at the end, these are also fabric scoped.

I think "id > 0" is fine here.

Reply:
They are also fabric scoped in terms of their keys, but they are not valid as things you might add specific endpoints to, right?  

@rcasallas-silabs, thank you.